latex/epoch.tex

```diff
@@ -77,6 +89,65 @@
 to assign a refund to each pool key in the set of pools scheduled for retirement
 at the epoch boundary.
 
+%%
+%% Figure - Functions for Stake Distribution
+%%
+\begin{figure}[htb]
+  \begin{align*}
+      & \fun{baseStake} \in \powerset{\TxOut} \to \powerset{(\HashKey \times \Coin)}
+      & \text{base address stake} \\
+      & \fun{baseStake}~{txouts} = \\
+      & ~~ \{(hk, c) \mid a\in\Addr_{base} \land (a, c) \in txouts \land \fun{stakeHK}~a = hk \}
```

Comment:
is it ok to call P(TxOut) txouts? It might be easy to confuse with the txouts tx, which gives a UTxO?

Reply:
yea, that could be confusing. the variables are in italics, and the functions are in mathsf, so you can tell the difference, but it is still probably confusing. maybe `outs`?

Reply:
i think that would work